src/course-outline/status-bar/StatusBar.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Button, Hyperlink, Stack } from '@edx/paragon';
+import { AppContext } from '@edx/frontend-platform/react';
+
+import { getPagePath } from '../../utils';
+import messages from './messages';
+
+const StatusBar = ({
+  statusBarData,
+  isLoading,
+  courseId,
+  openEnableHighlightsModal,
+}) => {
+  const intl = useIntl();
+  const { config } = useContext(AppContext);
+
+  const {
+    courseReleaseDate,
+    highlightsEnabledForMessaging,
+    highlightsDocUrl,
+    checklist,
+    isSelfPaced,
+  } = statusBarData;
+
+  const {
+    completedCourseLaunchChecks,
+    completedCourseBestPracticesChecks,
+    totalCourseLaunchChecks,
+    totalCourseBestPracticesChecks,
+  } = checklist;
+
+  const checkListTitle = `${completedCourseLaunchChecks + completedCourseBestPracticesChecks}/${totalCourseLaunchChecks + totalCourseBestPracticesChecks}`;
+
+  if (isLoading) {
+    // eslint-disable-next-line react/jsx-no-useless-fragment
+    return <></>;
+  }
+
+  return (
+    <Stack direction="horizontal" gap={3.5} className="outline-status-bar">
+      <div className="outline-status-bar__item">
+        <h5 className="h5">{intl.formatMessage(messages.startDateTitle)}</h5>
+        <Hyperlink
+          className="small"
+          destination={getPagePath(courseId, process.env.ENABLE_NEW_SCHEDULE_DETAILS_PAGE, 'settings/details#schedule')}
+          showLaunchIcon={false}
+        >
+          {courseReleaseDate}
+        </Hyperlink>
+      </div>
+      <div className="outline-status-bar__item">
+        <h5 className="h5">{intl.formatMessage(messages.pacingTypeTitle)}</h5>
+        <span className="small">
+          {isSelfPaced
+            ? intl.formatMessage(messages.pacingTypeSelfPaced)
+            : intl.formatMessage(messages.pacingTypeInstructorPaced)}
+        </span>
+      </div>
+      <div className="outline-status-bar__item">
+        <h5 className="h5">{intl.formatMessage(messages.checklistTitle)}</h5>
+        <Hyperlink
+          className="small"
+          destination={`${config.STUDIO_BASE_URL}/checklists/${courseId}`}
```

Comment:
optional: move destination out of return to `const ...destination = ` here, below and above.